docker-compose.override.yml

```diff
@@ -33,8 +33,12 @@ services:
       - /tmp
 
   assets:
-    command: npm run dev
     <<: *x-dev-defaults
+    # As we have mounted the app directory into the container and webpacker expects the
```

Comment:
Not that it's too important, but just for clarification, we don't use webpack/webpacker anymore. It was removed in [Replace webpacker #1081](https://github.com/tactilenews/100eyes/pull/1081). We are using **esbuild** to build JS and **prettier** for CSS.
It seems to be a simplification that Rails 7 shipped and Till found a way that we can already benefit from it in Rails 6.